### PR TITLE
Changing demo host

### LINKED ISSUE #107: Cuenta en revisión
Now, when we register new accounts in STP it has `Estado: A`, `observaciones: Cuenta en revisión.`.

In the responses they send:
```json
{
   "descripcion": "Cuenta en revisión.", 
   "id": 0
}
```

### LINKED ISSUE #116: Host para demo
El host para ambiente de pruebas cambió:
Old host: https://demo.stpmex.com:7024/speidemows/rest
New host: https://demo.stpmex.com:7024/speiws/rest

### PR DESCRIPTION
**Pull request by chiqeen03**
STP changed their demo host.

Old host: https://demo.stpmex.com:7024/speidemows/rest
New host: https://demo.stpmex.com:7024/speiws/rest

closes #107 
closes #116

### CODE REVIEW

File: stpmex/client.py [hunk 1]
```diff
@@ -29,8 +29,8 @@
 from .version import __version__ as client_version
 
 DEMO_HOST = 'https://demo.stpmex.com:7024'
-DEMO_BASE_URL = f'{DEMO_HOST}/speidemows/rest'
-DEMO_SOAP_URL = f'{DEMO_HOST}/speidemo/webservices/SpeiConsultaServices'
+DEMO_BASE_URL = f'{DEMO_HOST}/speiws/rest'
```
Comment (matin) [inline stpmex/client.py:32]: This is now redundant with `PROD_BASE_URL`. If the endpoints both both environments are the same, you should handle this differently

**Review by matin — CHANGES_REQUESTED** [review]
Only one change and bump the maintenance version of the library

**Comment by matin** [comment]
Rebase as well. 

While you're at it, you should incorporate the version changes proposed by dependabot

**Comment by matin** [comment]
Take a look at https://github.com/cuenca-mx/stpmex-python/pull/108 on how to handle the account revision change. Not sure why it's still in draft, but we might as well resolve it

**Comment by matin** [comment]
Bump the minor version since there's a breaking change from incorporating #118

File: stpmex/client.py [hunk 2]
```diff
@@ -154,7 +153,9 @@ def _raise_description_exc(resp: Dict) -> NoReturn:
     id = resp['id']
     desc = resp['descripcion']
 
-    if id == 1 and desc == 'Cuenta Duplicada':
+    if id == 0 and 'Cuenta en revisión' in desc:
+        ...
```
Comment (felipaoo) [inline stpmex/client.py:157]: Agrega comentario de porque motivo no se hace ninguna acción en este caso.

File: stpmex/client.py [hunk 3]
```diff
@@ -108,8 +108,7 @@ def _check_response(response: Response) -> None:
         resp = response.json()
         if isinstance(resp, dict):
             try:
-                if 'descripcionError' in resp['resultado']:
-                    _raise_description_error_exc(resp)
+                _raise_description_error_exc(resp)
```
Comment (felipaoo) [inline stpmex/client.py:111]: cambiar código anidado por algo más plano, ej: https://github.com/cuenca-mx/stpmex-python/pull/108/files#diff-b23372b5bc04ed97bbf3529676496f80R115-R120

**Review by felipaoo — CHANGES_REQUESTED** [review]

**Review by felipaoo — DISMISSED** [review]
Please add `closes #` in the main comment

**Review by felipaoo — APPROVED** [review]

**Pull request merged**